test-suite/Makefile

```diff
@@ -242,7 +242,7 @@ $(addsuffix .log,$(wildcard bugs/opened/*.v)): %.v.log: %.v
 	    echo "    $<...still active"; \
 	  else \
 	    echo $(log_failure); \
-	    echo "    $<...Error! (bug seems to be closed, please check)"; \
+	    echo "    $<...Error! (bug seems to be opened, please check)"; \
```

Comment:
I don't understand this change. If the bug was already opened, we want to know when it is (inadvertently) closed.

Reply:
Oh. I misunderstood the error message, then. Let me revert this.